smartsim/database/orchestrator.py

```diff
@@ -388,6 +396,15 @@ def _redis_exe(self) -> str:
     def _redis_conf(self) -> str:
         return CONFIG.database_conf
 
+    @property
+    def checkpoint_file(self) -> str:
+        """Get the path to the checkpoint file for this Orchestrator
+
+        :return: Path to the checkpoint file if it exists, otherwise a None
+        :rtype: str or None
```

Comment:
No need to add the rtype here I believe!

Reply:
`:rtype:` is gone!